guides/idea_tomcat_setup.md

```diff
@@ -0,0 +1,37 @@
+# IntelliJ IDEA Project Setup Guide
+
+Follow these detailed steps to set up your project in IntelliJ IDEA seamlessly:
+
+1. Build with Maven. Run the following commands from the command line:
+    - `mvn clean install -f dhis-2/pom.xml -DskipTests=true`
+    - `mvn clean install -f dhis-2/dhis-web/pom.xml -DskipTests=true`

```

Comment:
We do not have any more `dhis-web` modules.
This guide could be renamed to specify the exact versions of DHIS2 where it will work and the create a guide for master